src/lib/containers/entity_finder/details/configurations/EntityChildrenDetails.tsx

```diff
@@ -19,6 +19,13 @@ export const EntityChildrenDetails: React.FunctionComponent<EntityChildrenDetail
   const [sortDirection, setSortDirection] = useState<Direction>(Direction.ASC)
   const handleError = useErrorHandler()
 
+  const requestKey = {    
+    parentId: parentContainerId,
+    includeTotalChildCount: true,
+    includeTypes: sharedProps.visibleTypes,
+    sortBy: sortBy,
+    sortDirection: sortDirection,
+  }
```

Comment:
```suggestion
  const getChildrenInfiniteRequestObject = {    
    parentId: parentContainerId,
    includeTotalChildCount: true,
    includeTypes: sharedProps.visibleTypes,
    sortBy: sortBy,
    sortDirection: sortDirection,
  }
```

You might need to `useMemo` here, too?